client/my-sites/marketing/buttons/preview-buttons.jsx

```diff
@@ -124,11 +128,11 @@ class SharingButtonsPreviewButtons extends Component {
 		if ( 'official' === this.props.style ) {
 			// To show the more preview when rendering official style buttons,
 			// we request that the frame emit a show message with the offset
-			ReactDom.findDOMNode( this.refs.iframe ).contentWindow.postMessage( 'more-show', '*' );
+			this.previewIframe.current.contentWindow.postMessage( 'more-show', '*' );
 		} else {
 			// For custom styles, we can calculate the offset using the
 			// position of the rendered button
-			moreButton = ReactDom.findDOMNode( this.refs.moreButton );
+			moreButton = this.moreButton.current;
```

Comment:
Here `ReactDom.findDOMNode` can't be just removed, because `this.moreButton.current` is an instance of the `SharingButtonsPreviewButton`, not a DOM element. The component doesn't forward the ref to the inner element.